src/pages/Dashboard/scenes/Emails/EmailHistory.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useEffect, useState, useCallback } from 'react';
+import axios from 'axios';
+import { EMAILAPI_URL } from '../../../../constants';
+import LoadingSVG from '../../../../assets/svg/LoadingSVG';
+import { type Email } from '../../../../types';
+
+const EmailHistory: React.FC<{ refreshCount: number }> = ({ refreshCount }) => {
+  const [emails, setEmails] = useState<Email[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const fetchEmails = useCallback(() => {
+    setIsLoading(true);
+    setTimeout(() => {
+      axios
+        .get(`${EMAILAPI_URL}/api/v1/sent`)
+        .then((response) => {
+          setEmails(
+            Array.isArray(response.data.emails) ? response.data.emails : []
+          );
```

Comment:
The API should send [] if there's not data

Reply:
like this  ?
![image](https://github.com/sef-global/scholarx-frontend/assets/48531182/58a274e0-bf72-4846-8ecd-c38285b5ed23)

Reply:
yeah